docs/_docs/includes.md

```diff
@@ -55,35 +55,41 @@ You can also pass parameters to an include. For example, suppose you have a file
 
 ```liquid
 {% raw %}<div markdown="span" class="alert alert-info" role="alert">
-<i class="fa fa-info-circle"></i> <b>Note:</b> {{include.content}}
+<i class="fa fa-info-circle"></i> <b>Note:</b>
+{{ include.content }}
 </div>{% endraw %}
 ```
 
-The {% raw %}`{{include.content}}`{% endraw %} is a parameter gets populated when you call the include and specify a value for that parameter, like this:
+The `{% raw %}{{ include.content }}{% endraw %}` is a parameter gets populated when you call the include and specify a value for that parameter, like this:
```

Comment:
can you remind me what the misjudgement was? I simply forgot to put the raw tags inside the backticks. it renders fine.